.github/workflows/clang-tidy.yml

```diff
@@ -0,0 +1,34 @@
+name: clang-tidy-check
+on:
+  pull_request:
+    paths-ignore:
+      - '.editorconfig'
+      - '.gitattributes'
+      - '.github/*_TEMPLATE/**'
+      - '.github/workflows/localisation.yml'
+      - '.gitignore'
+      - '.vscode/**'
+
+jobs:
+  clang-tidy-check:
+    runs-on: ubuntu-latest
+    container: openrct2/openrct2-build:12-jammy
+    # This job requires a pull request event to run.
+    if: github.event_name == 'pull_request'
```

Comment:
Is this needed, since the job already applies only to on pull requests?

Reply:
No, ill remove it, was initially in the CI workflow.